README.md

```diff
@@ -1,48 +1,51 @@
-![parma logo](https://s3-ap-southeast-1.amazonaws.com/parma-img/parma_logo_mid.png)
-# parma
+![parma logo](https://cldup.com/8ULBQyooFe.png)
```

Comment:
Its the same logo but just resized it and cropped the empty space around it. I can give you the update file and host it again on AWS